lib/features.ts

```diff
@@ -88,3 +88,10 @@ export function isDPoPSupported () {
     hasTextEncoder() &&
     isWebCryptoSubtleSupported();
 }
+
+export function isIOS () {
```

Comment:
Some unit tests to cover the expected behavior of this utility would be beneficial

Reply:
added